network/common/src/main/java/org/apache/spark/network/server/TransportRequestHandler.java

```diff
@@ -82,10 +75,8 @@ public void exceptionCaught(Throwable cause) {
 
   @Override
   public void channelUnregistered() {
-    // Inform the StreamManager that these streams will no longer be read from.
-    for (long streamId : streamIds) {
-      streamManager.connectionTerminated(streamId);
-    }
+    // Inform the StreamManager that this channel is unregistered.
```

Comment:
nit: I think we can remove this comment, the new code is straightforward enough.